src/expressions/operator-expr.md

```diff
@@ -347,10 +347,16 @@ well as the following additional casts. Here `*T` means either `*const T` or
 | `*T` where `T: Sized` | Numeric type          | Pointer to address cast          |
 | Integer type          | `*V` where `V: Sized` | Address to pointer cast          |
 | `&[T; n]`             | `*const T`            | Array to pointer cast            |
-| [Function pointer](../types/function-pointer.md) | `*V` where `V: Sized` | Function pointer to pointer cast |
-| Function pointer      | Integer               | Function pointer to address cast |
+| [Function item]       | [Function pointer]    | Function item to function pointer cast |
+| [Function item]       | `*V` where `V: Sized` | Function item to pointer cast    |
+| [Function item]       | Integer               | Function item to address cast    |
+| [Function pointer]    | `*V` where `V: Sized` | Function pointer to pointer cast |
+| [Function pointer]    | Integer               | Function pointer to address cast |
 | Closure \*\*          | Function pointer      | Closure to function pointer cast |
 
+[Function pointer]: ../types/function-pointer.md
```

Comment:
These should be at the bottom of the page with the other link refs in alphabetical order.

Reply:
I agree about putting link at page bottom. However currently links order is not alphabetical.

Reply:
And sorting so will make the diff bigger.